apis/quarkus-common/src/test/java/io/stargate/sgv2/docsapi/api/common/security/HeaderBasedAuthenticationMechanismTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.stargate.sgv2.docsapi.api.common.security;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+
+import io.quarkus.test.junit.QuarkusTest;
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import org.junit.jupiter.api.Test;
+
+@QuarkusTest
+public class HeaderBasedAuthenticationMechanismTest {
```

Comment:
Currently failing with 404s, I haven't yet determined why.

Reply:
Fixed in https://github.com/stargate/stargate/commit/905f22e4873d656d105af38be426717a2de412c7

I guess the issue was that no controller exists in `src/`